src/test/findReadMeTest.ts

```diff
@@ -2,34 +2,43 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 import { findReadMe } from "../findReadMe"
-import * as assert from "assert"
+import * as vfs from "@ts-common/virtual-fs"
+
+const readmePaths = [
+    "https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/readme.md",
+    vfs.pathJoin(vfs.pathResolve("."), "readme.md"),
+]
+
+jest.spyOn(vfs, "exists").mockImplementation((filename: string)  => {
```

Comment:
I mocked those calls as for some reason they were taking for ever and was timeout the tests(12s if I increased timeout)

Reply:
Because we don't have separate integration test for this, I would propose to keep at least one test that tries real thing.

Reply:
As discussed, switching back to increased timeout for now as there is very few tests. If the amount of tests becomes larger we can look into seprating integration from unit tests `*.e2e.ts` for example